lib/customer.rb

```diff
@@ -0,0 +1,33 @@
+require_relative 'order'
+
+class Customer
+  include Menu
+  include Text
+
+  def check_menu
+    display_menu
+  end
+
+  def verify_and_pay(order, total_price)
+    if order.order_total(total_price) == true
```

Comment:
No need to check `== true` as anything other than `nil` or `false` will evaluate to true.

This includes `0`, bear in mind.